go-apps/meep-app-enablement/server/app-support/app-support.go

```diff
@@ -129,6 +129,9 @@ func Run() (err error) {
 
 // Stop - Stop APP support
 func Stop() (err error) {
+	// Flush all app-enablement instance data
```

Comment:
Is there a risk that _basekey_ is not initialized here and we flush the entire redis DB?
We should add a running flag to make sure we are running before stopping the microservice.

Reply:
baseKey is set in the init, so i believe it should have a value that is not "".... but i added a check just in case...